src/tools/cephfs/Dumper.cc

```diff
@@ -227,23 +227,54 @@ int Dumper::undump(const char *dump_file, bool force)
   char buf[HEADER_LEN];
   r = safe_read(fd, buf, sizeof(buf));
   if (r < 0) {
+    cout << "Error reading " << dump_file << std::endl;
```

Comment:
`derr`?

Reply:
We could change this to `derr` here as the current qa test doesn't check this error. Agree?

Reply:
Updated at this line.